site/utils/index.ts

```diff
@@ -25,6 +26,12 @@ export const redirectTo = (res: NextApiResponse | ServerResponse, location: stri
 export const getCsrfToken = (ctx: NextPageContext | NextPageContext): string =>
     (ctx.res as ResponseWithLocals)?.locals?.csrfToken ?? "";
 
+export const convertDateTimeToFormat = (dateOrTime: string, format: string) =>
+    dateOrTime && format ? dayjs(dateOrTime).format(format) : "";
+
+export const formatTime = (time: string) => (time ? time.slice(0, -2) + ":" + time.slice(-2) : "");
+
+export const splitCamelCaseToString = (s: string) => (s ? upperFirst(lowerCase(startCase(s))) : "");
```

Comment:
Basically the change I think we need to make is remove all the checks to see if the input is defined, because typescript already enforces it being defined.

Reply:
done